.circleci/config.yml

```
@@ -214,8 +214,7 @@ jobs:
           command: |
             set -x
             source /usr/local/etc/profile.d/conda.sh && conda activate python${PYTHON_VERSION}
-            conda install -v -y -c pytorch-nightly pytorch
-            conda install -v -y $(ls ~/workspace/torchtext*.tar.bz2)
+            conda install -v -y -c pytorch-nightly pytorch file://$HOME/workspace/conda-bld
```

```suggestion
            conda install -v -y -c pytorch-nightly -c file://$HOME/workspace/conda-bld torchtext
```

With the `file://` local repository it should automatically download and install `pytorch` since it's listed as a dependency.

Note this will actually only install the local package if the version we are installing locally is greater than the latest version found in the `pytorch-nightly` channel

---

> Note this will actually only install the local package if the version we are installing locally is greater than the latest version found in the pytorch-nightly channel

Can `torchtext` in `pytorch-nightly` channel have the same version as `local build`?

Instead of installing `pytorch` and `torchtext` at the same time, will installing them separately, enabling each channel separately make sure that local `torchtext` is installed? 

i.e.

```
conda install -v -y -c pytorch-nightly pytorch
conda install -v -y -c file://$HOME/workspace/conda-bld torchtext
```

---

Yes that could work, I only suggested doing it all in one command because of convenience.

> Can torchtext in pytorch-nightly channel have the same version as local build?

That's a good question, I'm not entirely sure how conda would prioritize certain channels over another. Doing the two step process is probably preferable if you're expecting them to potentially have the same versions.

---

Sounds good. Updated.